automation_tools/__init__.py

```diff
@@ -628,6 +628,37 @@ def setup_abrt():
     run('abrt-auto-reporting enabled')
 
 
+def setup_code_coverage():
+    """Task to setup code coverage on sat6.
+
+    Expects the following environment variables:
+
+        SITECUSTOMIZE_URL
+                The sitecustomize file to launch the coverage.
+        COVERAGERC_URL
+                The coverage configuration file.
+
+    """
+    coverage_file = '/etc/coverage/.coveragerc'
```

Comment:
I think it is ok on leaving here, having this value on a variable improve the code readability and having them here will avoid need to check on other part of the code for the value. 

I would suggest moving to a constants place if this value was used in more than one function.